breathecode/events/models.py

```diff
@@ -182,6 +182,8 @@ def __init__(self, *args, **kwargs):
                             validators=[validate_language_code])
     currency = models.CharField(max_length=3, choices=CURRENCIES, default=USD, blank=True)
     tags = models.CharField(max_length=100, default='', blank=True)
+    free_for_all = models.BooleanField(
```

Comment:
There have so many undefined variables
https://github.com/breatheco-de/apiv2/actions/runs/6882340259/job/18720662409?pr=1163

Reply:
The undefined variables are not related to the code I added. Maybe they should be fixed in a different issue

Reply:
It's solved now, you were right